extensions/amp-consent/0.1/consent-state-manager.js

```diff
@@ -316,6 +316,7 @@ export class ConsentInstance {
     });
     cidPromise.then(userId => {
       const request = /** @type {!JsonObject} */ ({
+        // Unfortunately we need to keep the name to be backward compatible
```

Comment:
@torch2424 @lannka I do want to get your idea on this. 

I changed the `config['consentInstanceId']` to `config['storageKey']` because storageKey describe the purpose of the field better. But I can't change the `consentInstanceId` in request. 

I am thinking of two options
1. Keep the 'consentInstanceId', and document that it will be served as the storageKey
2. Migrate to 'storageKey', but add an additional 'storageKey' field along with all requests. 

I'm leaning to option 2, but keeping two names of the same instance could be confusing. Let me know WDYT?

Reply:
Discussed offline. Option 1 seems like the better route. Having two keys represent the same thing is more of a code smell, then something with a mediocre name 😄 (Which in my opinion, `consentInstanceId` isn't really a bad name 👍 )

Reply:
😄Agreed. I switched `storagekey` back to `consentInstanceId`.